pywbemtools/pywbemcli/_common.py

```diff
@@ -1004,6 +1004,63 @@ def get_wbemurikeys(obj):
     return wbem_uri_keys
 
 
+class NoCaseList(object):
+    """
+    This class is a dervivative of lists the simplifies working with
+    some of the CIM object attributes that are case insensitive such as
+    class names, roles, etc.
```

Comment:
I find the term "derivative of lists" confusing. The class is not derived from the Python `list` type, nor is its behavior compatible to a Python list.

Reply:
Removed that confusing statement completly.  Just says "This class simplifies working with lists of strings that are to be tested  case-insenstive." DONE